source/Genome.cpp

```diff
@@ -140,13 +140,13 @@ void Genome::genomeLoad(){//allocate and load Genome
         errOut << "EXITING because of FATAL ERROR: read no value for the versionGenome parameter from genomeParameters.txt file\n";
         errOut << "SOLUTION: please re-generate genome from scratch with the latest version of STAR\n";
         exitWithError(errOut.str(),std::cerr, P.inOut->logMain, EXIT_CODE_GENOME_FILES, P);
-    } else if (P1.versionGenome == P.versionGenome) {//
+    } else if (P1.versionGenome == P.versionGenome || (!P.quant.yes && P1.versionGenome == "20201")) {//
```

Comment:
Could you please add a comment here to indicate that `versionGenome` of `20201` corresponds to STAR 2.5.1b? Otherwise LGTM. Thanks, Lance!

Reply:
or 2.7.1a rather re

> 20201 jumps straight to 2.7.1a, so these are the only supported versions at present.